src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -1612,12 +1606,12 @@ private unsafe void WriteEventRaw(
 
         // FrameworkEventSource is on the startup path for the framework, so we have this internal overload that it can use
```

Comment:
This comment should be deleted and replaced with tripple slash comment that will be used to populate the docs.